### ISSUE TITLE
glbuoys problem with Android

### ISSUE THREAD

**Issue by beckypearson** [glos/user_queries#26]
'Harold Reitsema' via DMAC | 'Harold Reitsema' via DMAC

I have been able to open the page for buoy 45029 without problem until the last week and now it gives me just a blank page. It opens fine with Chrome. I have a Motorola Droid through Verizon.Thanks. HaroldSent from my Verizon Wireless 4G LTE DROID
--

Sorry you are having issues Harold.  Can you help us troubleshoot this a bit and tell us what the exact link is you are using (or which of our portal products you are attempting to access it from?)
 
Kathy
_____________________________________
Kathy Koch
Data Curator
Hi, Kathy,
 
----

Thanks for helping with this!
 
I go to glbuoys.glos.us
Then I go the then drop-down menu and choose “Lake Michigan” and “45029”
I get a page with the glos and Great Lakes Buoys header but it is blank.
I also can’t get there with the bookmark that I have used for over a year. It is long so I may not have copied it perfectly from my smartphone but it is
Glbuoys.glos.us/show_pic.php?station=45029&meas=WVHGT&uom=E&time-diff=-4&time_level=EDT
That link brings up a page that says “sorry, we couldn’t find the page you were looking for”
It then gives me a link eith EDT in it to try, but that gives me the same page back again.
 
Best,
 
Harold
---

Looks like the redirect from our old site to the new is giving you some apples vs. oranges for the parameters you are looking for.
 
This is the buoy link at our new site > http://glbuoys.glos.us/45029
 
In order to get the exact page you had before on the legacy website (which is still available), you would change the first part of your url to http://uglos.mtu.edu
 
In other words, this would work >
 
http://uglos.mtu.edu/show_pic.php?station=45029&meas=WVHGT&uom=E&time-diff=-4&time_level=EDT
 
 
Kathy
